cEP-0020.md

```diff
@@ -0,0 +1,543 @@
+  - Privileges:
+    - Mark your own PR as wip
+- Level-V:
+  - Name: Cool
+  - Points_required = 100
+  - Privileges:
+    - Mark a PR by another newcomer as wip
+    - Get assigned to difficulty medium issue
+- Level-VII:
+  - Name: Expert
+  - Points_required = 200
+  - Privileges
+    - Assign newcomers to difficulty newcomer issue
+    - Assign newcomers to difficulty low issue
+- Level-VIII:
+  - Name: Legend
+  - Points_required = 350
+  - Privileges
+    - Invite newcomers to org
+    - Assign newcomers to difficulty medium issue
+    - Permission to `ack` and `unack` on the pull requests
+
+Implementation of privileges is beyond scope of the
+gamification system. The gamification system will merely
+expose a list of privileges that each person has, and other
+systems are expected to adapt to use these privileges.
+E.g.: [corobo](https://github.com/coala/corobo)
+
+While implementing privileges, a first manual check should
+be done for testing purpose, and then automated only if there
+is no serious risk or there is a good risk reaction mechanism
+in place.
+
+### Badges
+
+Another way of explicitly nudging a newcomer to action is to
+award badges for completing tasks. Badges enable newcomers to
+follow their performance and compare to other newcomers.
+Badges will be awarded after performing certain activities by
+the newcomers:
+
+- Badge-I:
+  - Name: The bug finder
+  - Details: The one who find bugs in the existing codebase
+  - Activities:
+    - Create at least one issue which labelled as bug
+- Badge-II:
+  - Name: The bear hunter
+  - Details: The one who create issues about new bears
+  - Activities:
+    - Create at least one bear proposal issue
+- Badge-III:
+  - Name: The bear writer
+  - Details: The one who write new bears
+  - Activities:
+    - Merged at least 2 bear proposal PRs
+- Badge-IV:
+  - Name: The helper
+  - Details: The one who answers every newcomer questions on Gitter
+  - Activities:
+    - At least 20 answers given on coala Gitter rooms
+- Badge-V:
+  - Name: The coala Developer
+  - Details: The ones who are in the coala developers team
+  - Activities:
+    - Introduce `.coafile` in user own repository
+    - Merge a difficulty/newcomer Pull Request
+    - Review at least a difficulty/newcomer Pull Request
+    - Merge a difficulty/low Pull Request
+    - Review at least a difficulty/low or higher Pull Request
+
+So, for becoming a developer at coala, a newcomer must complete
+all the activities mentioned in the `coala Developer` badge.
+But in case some newcomers are not able to become a developer
+at coala, they could take away the other badges.
+
+Names and metrics of tracked activities will also be shown
+on the leaderboard along with the number of times they
+have been done. That will help maintainers to check if an
+automatically awarded badge is correct or not. So, in case
+of any doubt on the newcomer, a maintainer can do the
+manual check and not promote them to developers team.
```

Comment:
> So, in case ..

You talk about the developers team in the next paragraph -- dont mention it here.  move anything in this sentence to the next paragraph.